apps/preview/next/pages/showcases/Filters/Brand.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfAccordionItem, SfCheckbox, SfCounter, SfIconChevronLeft, SfListItem } from '@storefront-ui/react';
+import { useState } from 'react';
+import classNames from 'classnames';
+
+const details = [
+  { id: 'b1', label: 'Conroy', value: 'conroy', counter: 10 },
+  { id: 'b2', label: 'Goyette', value: 'goyette', counter: 100 },
+  { id: 'b3', label: 'Ledner & Ward', value: 'lednerward', counter: 0 },
+  { id: 'b4', label: 'Pacocha', value: 'pacocha', counter: 3 },
+];
+
+export default function BrandFilter() {
+  const [opened, setOpened] = useState(true);
```

Comment:
This is perfect use case for [`useDisclosure`](https://docs.storefrontui.io/v2/vue/hooks/useDisclosure.html).

Reply:
@dkacper Do you think it should be refactored in all filter blocks in the scope of this PR or in a separate task?